authz.go

```diff
@@ -163,7 +163,7 @@ func (a *authorizer) authorize(ctx context.Context, sr authz.SubjectAccessReview
 		if err != nil {
 			return deny(NewAuthzError(err, internal), true)
 		}
-		granted, err = client.authorize(ctx, principal, check)
+		granted, err = client.authorize(ctx, principal, check, a.AuthorizationConfig.Config)
```

Comment:
As @prabushyam suggested, let's not pass the config here but call authorize above it so we can compare the return values.